webapp-django/crashstats/signature/static/signature/js/signature_tab_summary.js

```diff
@@ -18,3 +18,14 @@ SignatureReport.SummaryTab = function(tabName) {
 };
 
 SignatureReport.SummaryTab.prototype = SignatureReport.inherit(SignatureReport.Tab.prototype);
+
+SignatureReport.SummaryTab.prototype.onAjaxSuccess = function (contentElement, data) {
+    SignatureReport.Tab.prototype.onAjaxSuccess.apply(this, arguments);
+
+    $(contentElement).on('click', '.toggle-visible', function (e) {
+        e.preventDefault();
+        var elt = $(this);
+        $('.content', elt.parent()).toggle();
+        $('.options', elt).toggleClass('hide').toggleClass('show');
```

Comment:
Remove the `show` and just make it default to show in the CSS. The only toggle on hide. 
It's like saying "It's hidden and not visible" & "It's visible and not hidden" when it's simpler to just say "It's hidden" & "It's not hidden".